instrumentation/httpurlconnection/testing/src/main/java/io/opentelemetry/instrumentation/library/httpurlconnection/HttpUrlConnectionTestUtil.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.library.httpurlconnection;
+
+import android.util.Log;
```

Comment:
If you remove the log dependency, this can be part of the `test-common` I guess?
Its a plain Java file that can be reused across different modules

Reply:
These need to be here as we're instrumenting these with our httpurlconnection bytebuddy agent so we can test the same.